src/main/java/mpern/sap/commerce/build/util/Extension.java

```diff
@@ -1,20 +1,43 @@
 package mpern.sap.commerce.build.util;
 
 import java.nio.file.Path;
+import java.util.List;
 import java.util.Objects;
 import java.util.StringJoiner;
 
 public class Extension {
+
     public final String name;
     public final Path directory;
```

Comment:
I would suggest we delete this in the future, it is quite platform specific and used mostly for logging in the ccv2 tasks. We can use the relativeLocation